testnets/titantestnet/chain.json

```diff
@@ -0,0 +1,163 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "titantestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "website": "https://tokenize.exchange/",
+  "pretty_name": "Titan Testnet",
+  "chain_id": "titan_18889-1",
+  "bech32_prefix": "titan",
+  "node_home": "$HOME/.titand",
+  "daemon_name": "titand",
+  "key_algos": [
+    "ethsecp256k1"
+  ],
+  "extra_codecs": [
+    "ethermint"
+  ],
+  "slip44": 60,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "atkx",
+        "fixed_min_gas_price": 100000000000,
+        "low_gas_price": 100000000000,
+        "average_gas_price": 110000000000,
+        "high_gas_price": 200000000000
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "atkx"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/tokenize-titan/titan",
+    "recommended_version": "v1.0.0",
+    "compatible_versions": [
```

Comment:
Can you please add these version details (the full list of compatible versions, cosmos sdk version, consensus, ibc go version, cosmwasm , etc...) into the object inside the versions array as well?

Reply:
Thank you. I just correct version information in `versions` array.